.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
@Jules-Bertholet Is this needed?  We could also at `.vscode/`, `.vim/` to `.gitignore`